controllers/clustermodule_reconciler.go

```diff
@@ -166,28 +170,34 @@ func (r Reconciler) Reconcile(clusterCtx *capvcontext.ClusterContext) (reconcile
 }
 
 func (r Reconciler) toAffinityInput(ctx context.Context, obj client.Object) []reconcile.Request {
+	log := ctrl.LoggerFrom(ctx)
+
 	cluster, err := util.GetClusterFromMetadata(ctx, r.Client, metav1.ObjectMeta{
 		Namespace:       obj.GetNamespace(),
 		Labels:          obj.GetLabels(),
 		OwnerReferences: obj.GetOwnerReferences(),
 	})
 	if err != nil {
-		r.Logger.Error(err, "failed to get owner cluster")
+		log.Error(err, "failed to get owner cluster")
 		return nil
 	}
+	log = log.WithValues("Cluster", klog.KObj(cluster))
```

Comment:
Nit: Use consistent casing for log keys. E.g. you have `Cluster` here, but `targetObjectName` above.

Reply:
Thx good point.

The conventions are a bit messy unfortunately. Upstream is using `targetObjectName`, `pod`, ... . We tried to do this in controller-runtime but there is simply no way to generate the right key for a CRD because there is no way to tell how to lower-case a kind correctly :). This is why we ended up with `Cluster`, `VSphereCluster`, etc. (we just use the kind directly).

Based on that we can either only use upper-case for kind and only be inconsistent with upstream there or we can do this consistently and be in-consistent with k/k everywhere :D (I think I would go with `Pod`, `Cluster`, `TargetObjectName`)

Reply:
I'll move this to a separate issue. I don't want to violate upstream conventions across the code base without a discussion

Reply:
Issue: https://github.com/kubernetes-sigs/cluster-api/issues/9447